src/main/java/com/targomo/client/Constants.java

```diff
@@ -100,6 +100,9 @@
     public static final String MULTIGRAPH_AGGREGATION_MAX_RESULT_VALUE_RATIO            = "maxResultValueRatio";
     public static final String MULTIGRAPH_AGGREGATION_MAX_RESULT_VALUE                  = "maxResultValue";
     public static final String MULTIGRAPH_AGGREGATION_FILTER_VALUES_FOR_SOURCE_ORIGINS  = "filterValuesForSourceOrigins";
+    public static final String MULTIGRAPH_AGGREGATION_SOURCE_PARAMETERS	        		= "sourceParameters";
+    public static final String MULTIGRAPH_AGGREGATION_FACTOR	        		        = "factor";
+    public static final String MULTIGRAPH_PREAGGREGATION_PIPELINE						= "preAggregationPipeline";
```

Comment:
please always use 4 spaces instead of tab - otherwise we get formation issues (like above)